### DISCUSSION TITLE
Adding event flags help

### DISCUSSION THREAD

**Discussion by alvv-z** [kwsch/PKHeX#3948] [Q&A] [answered]
Hi, I wanted to add some new event flags for bw and bw2 via pull request, however I'm not sure what the `r`, `s`, etc. mean in the second column in [flags_bw_en.txt](https://github.com/kwsch/PKHeX/blob/master/PKHeX.Core/Resources/text/script/gen5/flags_bw_en.txt). Is there any place I can look stuff like this up?

**Comment by sora10pls** [comment, marked as answer]
https://github.com/kwsch/PKHeX/blob/96fa89188a306d032aae26811e955d529b9d67f5/PKHeX.Core/Editing/Saves/Editors/EventOld/NamedEventType.cs#L29-L49

**Reply by alvv-z** [reply]
Thank you very much!
